src/main/java/com/google/firebase/database/tubesock/ThreadConfig.java

```diff
@@ -32,10 +37,25 @@ public ThreadConfig(ThreadInitializer threadInitializer, ThreadFactory threadFac
     this.threadFactory = checkNotNull(threadFactory);
   }
 
+  /**
+   * The {@link ThreadInitializer} implementation to be used for further configuring already
+   * created threads. In particular, this instance is used to rename the created threads, and turn
```

Comment:
This suggests that it would work with threads that were created way back in the days. This class is used to configure newly created threads, while the factory is used to create new threads.

Reply:
Updated the text.